src/Operations.jl

```diff
@@ -157,16 +164,16 @@ function deps_graph(env::EnvCache, uuid_to_name::Dict{UUID,String}, reqs::Requir
 
                 union!(all_versions_u, versions)
 
-                for (vr,dd) in deps_data
-                    all_deps_u_vr = get_or_make!(all_deps_u, vr)
+                for (ver, dd) in deps_data
```

Comment:
Here and below, `vr` stood for "version range", so I would either revert these changes or spell it out.

Reply:
Reverted this.